examples/llama/main.zig

```diff
@@ -86,26 +87,44 @@ pub fn generateText(
             decode_progress += output_freq;
             std.debug.print("{s}", .{output.items[n..]});
             tracer.frameEnd(frame_id, try std.fmt.bufPrintZ(tracer_buffer, "Decoded token {}/{} : {s}", .{ i + 1, output_tokens_len, output.items[n..] }));
+
+            // These are almost certainly not the most optimal way to detect an EOS token.
+            // Do tinyLlama or openLlama use different tokenizers? Can we get these eos values automatically?
+            if (std.mem.indexOfScalar(i32, token_buffer[decode_progress..], 128001)) |_| {
```

Comment:
ok let's keep this, since the demo is mostly made for llama model family it's fine to hardocode llama eos tokens.

can you use though `std.mem.indexOfAny` to match all 3 eos tokens in one go ?